website/src/components/Features.tsx

```diff
@@ -35,7 +36,7 @@ export function Features() {
         <div className="relative lg:col-span-5 lg:-mr-8 xl:absolute xl:inset-0 xl:left-1/2 xl:mr-0">
 
           <Image
-            alt="Picture of a sample email. Each news has a title and a brief descritpion made by the AI"
+            alt={t('altImg')}
```

Comment:
You should scope your translations  with the page. In this exemple, we don't know about which  alt you are talking about.
Also If you have multiple images, the naming will be complex. 
It's even more important with the translation file. For now you don't have a lot of translations but the translations files will keep growing and it will become unmaintainable.